evoting/service/service.go

```diff
@@ -245,7 +245,8 @@ func (s *Service) GetElections(req *evoting.GetElections) (*evoting.GetElections
 		if err != nil {
 			return nil, err
 		}
-		if election.IsUser(req.User) || election.IsCreator(req.User) {
+		if (election.IsUser(req.User) || election.IsCreator(req.User)) &&
```

Comment:
This if would be easier to read if the two sides of the && were two nested if's:
```
if election.IsUser(req.User) || election.IsCreator(req.User) {
  if req.Stage == 0 || req.Stage == election.Stage {
    // append...
  }
}
```
The two checks are checking different things, and as such each deserves it's own line, and maybe one or both deserve comments to explain subtleties.